{"message":"FATAL ERROR: jwt private key not defined","level":"error"}
{"message":"FATAL ERROR: jwt private key not defined","level":"error"}
{"message":"FATAL ERROR: jwt private key not defined","level":"error"}
{"message":"FATAL ERROR: jwt private key not defined","level":"error"}
{"message":"FATAL ERROR: jwt private key not defined","level":"error"}
{"message":"FATAL ERROR: jwt private key not defined","level":"error"}
{"message":"FATAL ERROR: jwt private key not defined","level":"error"}
{"level":"error","message":"POPL POPL","stack":"Error: POPL\n    at Object.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/index.js:61:7)\n    at Module._compile (internal/modules/cjs/loader.js:1137:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)\n    at Module.load (internal/modules/cjs/loader.js:985:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:878:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)\n    at internal/main/run_main_module.js:17:47"}
{"level":"error","message":"er er","stack":"Error: er\n    at Object.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/index.js:66:16)\n    at Module._compile (internal/modules/cjs/loader.js:1137:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)\n    at Module.load (internal/modules/cjs/loader.js:985:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:878:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)\n    at internal/main/run_main_module.js:17:47"}
{"name":"MongoNetworkError","level":"error","message":"failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}] failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}]","stack":"MongoNetworkError: failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}]\n    at Pool.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/topologies/server.js:438:11)\n    at Pool.emit (events.js:315:20)\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/pool.js:562:14\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/pool.js:995:11\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:31:7\n    at callback (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:264:5)\n    at Socket.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:294:7)\n    at Object.onceWrapper (events.js:422:26)\n    at Socket.emit (events.js:315:20)\n    at emitErrorNT (internal/streams/destroy.js:92:8)\n    at emitErrorAndCloseNT (internal/streams/destroy.js:60:3)\n    at processTicksAndRejections (internal/process/task_queues.js:84:21)"}
{"name":"MongoNetworkError","level":"error","message":"failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}] failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}]","stack":"MongoNetworkError: failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}]\n    at Pool.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/topologies/server.js:438:11)\n    at Pool.emit (events.js:315:20)\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/pool.js:562:14\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/pool.js:995:11\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:31:7\n    at callback (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:264:5)\n    at Socket.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:294:7)\n    at Object.onceWrapper (events.js:422:26)\n    at Socket.emit (events.js:315:20)\n    at emitErrorNT (internal/streams/destroy.js:92:8)\n    at emitErrorAndCloseNT (internal/streams/destroy.js:60:3)\n    at processTicksAndRejections (internal/process/task_queues.js:84:21)"}
{"name":"MongoNetworkError","level":"error","message":"failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}] failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}]","stack":"MongoNetworkError: failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}]\n    at Pool.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/topologies/server.js:438:11)\n    at Pool.emit (events.js:315:20)\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/pool.js:562:14\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/pool.js:995:11\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:31:7\n    at callback (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:264:5)\n    at Socket.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:294:7)\n    at Object.onceWrapper (events.js:422:26)\n    at Socket.emit (events.js:315:20)\n    at emitErrorNT (internal/streams/destroy.js:92:8)\n    at emitErrorAndCloseNT (internal/streams/destroy.js:60:3)\n    at processTicksAndRejections (internal/process/task_queues.js:84:21)"}
{"name":"MongoNetworkError","level":"error","message":"failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}] failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}]","stack":"MongoNetworkError: failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}]\n    at Pool.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/topologies/server.js:438:11)\n    at Pool.emit (events.js:315:20)\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/pool.js:562:14\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/pool.js:995:11\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:31:7\n    at callback (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:264:5)\n    at Socket.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:294:7)\n    at Object.onceWrapper (events.js:422:26)\n    at Socket.emit (events.js:315:20)\n    at emitErrorNT (internal/streams/destroy.js:92:8)\n    at emitErrorAndCloseNT (internal/streams/destroy.js:60:3)\n    at processTicksAndRejections (internal/process/task_queues.js:84:21)"}
{"name":"MongoNetworkError","level":"error","message":"failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}] failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}]","stack":"MongoNetworkError: failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}]\n    at Pool.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/topologies/server.js:438:11)\n    at Pool.emit (events.js:315:20)\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/pool.js:562:14\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/pool.js:995:11\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:31:7\n    at callback (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:264:5)\n    at Socket.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:294:7)\n    at Object.onceWrapper (events.js:422:26)\n    at Socket.emit (events.js:315:20)\n    at emitErrorNT (internal/streams/destroy.js:92:8)\n    at emitErrorAndCloseNT (internal/streams/destroy.js:60:3)\n    at processTicksAndRejections (internal/process/task_queues.js:84:21)"}
{"name":"MongoNetworkError","level":"error","message":"failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}] failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}]","stack":"MongoNetworkError: failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}]\n    at Pool.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/topologies/server.js:438:11)\n    at Pool.emit (events.js:315:20)\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/pool.js:562:14\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/pool.js:995:11\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:31:7\n    at callback (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:264:5)\n    at Socket.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:294:7)\n    at Object.onceWrapper (events.js:422:26)\n    at Socket.emit (events.js:315:20)\n    at emitErrorNT (internal/streams/destroy.js:92:8)\n    at emitErrorAndCloseNT (internal/streams/destroy.js:60:3)\n    at processTicksAndRejections (internal/process/task_queues.js:84:21)"}
{"name":"MongoNetworkError","level":"error","message":"failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}] failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}]","stack":"MongoNetworkError: failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}]\n    at Pool.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/topologies/server.js:438:11)\n    at Pool.emit (events.js:315:20)\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/pool.js:562:14\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/pool.js:995:11\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:31:7\n    at callback (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:264:5)\n    at Socket.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:294:7)\n    at Object.onceWrapper (events.js:422:26)\n    at Socket.emit (events.js:315:20)\n    at emitErrorNT (internal/streams/destroy.js:92:8)\n    at emitErrorAndCloseNT (internal/streams/destroy.js:60:3)\n    at processTicksAndRejections (internal/process/task_queues.js:84:21)"}
{"level":"error","message":"POPL POPL","stack":"Error: POPL\n    at Object.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/index.js:21:7)\n    at Module._compile (internal/modules/cjs/loader.js:1137:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)\n    at Module.load (internal/modules/cjs/loader.js:985:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:878:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)\n    at internal/main/run_main_module.js:17:47"}
{"level":"error","message":"er er","stack":"Error: er\n    at module.exports (/home/abdulloooh/bin/coding/learning/node/vidly/startup/startup.js:18:18)\n    at Object.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/index.js:3:29)\n    at Module._compile (internal/modules/cjs/loader.js:1137:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)\n    at Module.load (internal/modules/cjs/loader.js:985:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:878:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)\n    at internal/main/run_main_module.js:17:47"}
{"name":"MongoNetworkError","level":"error","message":"failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}] failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}]","stack":"MongoNetworkError: failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}]\n    at Pool.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/topologies/server.js:438:11)\n    at Pool.emit (events.js:315:20)\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/pool.js:562:14\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/pool.js:995:11\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:31:7\n    at callback (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:264:5)\n    at Socket.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:294:7)\n    at Object.onceWrapper (events.js:422:26)\n    at Socket.emit (events.js:315:20)\n    at emitErrorNT (internal/streams/destroy.js:92:8)\n    at emitErrorAndCloseNT (internal/streams/destroy.js:60:3)\n    at processTicksAndRejections (internal/process/task_queues.js:84:21)"}
{"name":"MongoNetworkError","level":"error","message":"failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}] failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}]","stack":"MongoNetworkError: failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}]\n    at Pool.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/topologies/server.js:438:11)\n    at Pool.emit (events.js:315:20)\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/pool.js:562:14\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/pool.js:995:11\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:31:7\n    at callback (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:264:5)\n    at Socket.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:294:7)\n    at Object.onceWrapper (events.js:422:26)\n    at Socket.emit (events.js:315:20)\n    at emitErrorNT (internal/streams/destroy.js:92:8)\n    at emitErrorAndCloseNT (internal/streams/destroy.js:60:3)\n    at processTicksAndRejections (internal/process/task_queues.js:84:21)"}
{"name":"MongoNetworkError","level":"error","message":"failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}] failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}]","stack":"MongoNetworkError: failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}]\n    at Pool.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/topologies/server.js:438:11)\n    at Pool.emit (events.js:315:20)\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/pool.js:562:14\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/pool.js:995:11\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:31:7\n    at callback (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:264:5)\n    at Socket.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:294:7)\n    at Object.onceWrapper (events.js:422:26)\n    at Socket.emit (events.js:315:20)\n    at emitErrorNT (internal/streams/destroy.js:92:8)\n    at emitErrorAndCloseNT (internal/streams/destroy.js:60:3)\n    at processTicksAndRejections (internal/process/task_queues.js:84:21)"}
{"name":"MongoNetworkError","level":"error","message":"failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}] failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}]","stack":"MongoNetworkError: failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1141:16) {\n  name: 'MongoNetworkError'\n}]\n    at Pool.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/topologies/server.js:438:11)\n    at Pool.emit (events.js:315:20)\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/pool.js:562:14\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/pool.js:995:11\n    at /home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:31:7\n    at callback (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:264:5)\n    at Socket.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/node_modules/mongodb/lib/core/connection/connect.js:294:7)\n    at Object.onceWrapper (events.js:422:26)\n    at Socket.emit (events.js:315:20)\n    at emitErrorNT (internal/streams/destroy.js:92:8)\n    at emitErrorAndCloseNT (internal/streams/destroy.js:60:3)\n    at processTicksAndRejections (internal/process/task_queues.js:84:21)"}
{"level":"error","message":"POPL POPL","stack":"Error: POPL\n    at module.exports (/home/abdulloooh/bin/coding/learning/node/vidly/startup/startup.js:16:9)\n    at Object.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/index.js:3:29)\n    at Module._compile (internal/modules/cjs/loader.js:1137:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)\n    at Module.load (internal/modules/cjs/loader.js:985:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:878:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)\n    at internal/main/run_main_module.js:17:47"}
{"level":"error","message":"Unexpected token '.' Unexpected token '.'","stack":"/home/abdulloooh/bin/coding/learning/node/vidly/startup/db.js:11\n  .then(() => console.log(\"Connected to mongodb successfully\"));\n  ^\n\nSyntaxError: Unexpected token '.'\n    at wrapSafe (internal/modules/cjs/loader.js:1053:16)\n    at Module._compile (internal/modules/cjs/loader.js:1101:27)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)\n    at Module.load (internal/modules/cjs/loader.js:985:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:878:14)\n    at Module.require (internal/modules/cjs/loader.js:1025:19)\n    at require (internal/modules/cjs/helpers.js:72:18)\n    at module.exports (/home/abdulloooh/bin/coding/learning/node/vidly/startup/startup.js:8:3)\n    at Object.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/index.js:3:29)\n    at Module._compile (internal/modules/cjs/loader.js:1137:30)"}
{"level":"error","message":"Unexpected token '.' Unexpected token '.'","stack":"/home/abdulloooh/bin/coding/learning/node/vidly/startup/db.js:11\n  .then(() => console.log(\"Connected to mongodb successfully\"));\n  ^\n\nSyntaxError: Unexpected token '.'\n    at wrapSafe (internal/modules/cjs/loader.js:1053:16)\n    at Module._compile (internal/modules/cjs/loader.js:1101:27)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)\n    at Module.load (internal/modules/cjs/loader.js:985:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:878:14)\n    at Module.require (internal/modules/cjs/loader.js:1025:19)\n    at require (internal/modules/cjs/helpers.js:72:18)\n    at module.exports (/home/abdulloooh/bin/coding/learning/node/vidly/startup/startup.js:8:3)\n    at Object.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/index.js:3:29)\n    at Module._compile (internal/modules/cjs/loader.js:1137:30)"}
{"level":"error","message":"winston is not defined winston is not defined","stack":"ReferenceError: winston is not defined\n    at Server.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/index.js:6:32)\n    at Object.onceWrapper (events.js:421:28)\n    at Server.emit (events.js:315:20)\n    at emitListeningNT (net.js:1347:10)\n    at processTicksAndRejections (internal/process/task_queues.js:83:21)"}
{"message":"Listening on port 3002","level":"info"}
{"message":"Listening on port 3002","level":"info"}
{"message":"Listening on port 3002","level":"info"}
{"message":"Connected to mongodb successfully","level":"info"}
{"message":"Listening on port 3002","level":"info"}
{"message":"Connected to mongodb successfully","level":"info"}
{"message":"Listening on port 3002","level":"info"}
{"message":"Connected to mongodb successfully","level":"info"}
{"message":"Listening on port 3002","level":"info"}
{"message":"Connected to mongodb successfully","level":"info"}
{"message":"Listening on port 3002","level":"info"}
{"message":"Connected to mongodb successfully","level":"info"}
{"level":"error","message":"FATAL ERROR: jwt private key not defined FATAL ERROR: jwt private key not defined","stack":"Error: FATAL ERROR: jwt private key not defined\n    at module.exports (/home/abdulloooh/bin/coding/learning/node/vidly/startup/startup.js:12:11)\n    at Object.<anonymous> (/home/abdulloooh/bin/coding/learning/node/vidly/index.js:4:29)\n    at Module._compile (internal/modules/cjs/loader.js:1137:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)\n    at Module.load (internal/modules/cjs/loader.js:985:32)\n    at Function.Module._load (internal/modules/cjs/loader.js:878:14)\n    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)\n    at internal/main/run_main_module.js:17:47"}
{"message":"Connected to mongodb successfully","level":"info"}
